docker/dirigible/templates/Dockerfile.template

```diff
@@ -0,0 +1,9 @@
+FROM dirigiblelabs/dirigible-all AS build
+
+FROM {{ .From }}
+
```

Comment:
Can you please add an env var here: `C2D_RELEASE` and reference to versions.yaml as here: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/docker/activemq/templates/Dockerfile.template#L13

Reply:
Added! Can you check, please! Is it ok?